### ISSUE TITLE
Scope of Simple Edge Discovery for Fall24 CAMARA release

### ISSUE THREAD

**Issue by Kevsy** [camaraproject/SimpleEdgeDiscovery#3]
This issue shall define which issues are within scope of  the API of the Simple Edge Discovery Sub Project for the Fall24 release, and will be regularly updated based on the decision within the EdgeCloud meetings.
 
Resource model name/structure changes
 
* camaraproject/EdgeCloud/issues/226
* camaraproject/EdgeCloud/issues/210
* camaraproject/EdgeCloud/issues/209
* * camaraproject/EdgeCloud/issues/163

Commonalities

* camaraproject/EdgeCloud/issues/225
* camaraproject/EdgeCloud/issues/202
* camaraproject/EdgeCloud/issues/193

Documentation

* camaraproject/EdgeCloud/issues/207

API Release Criteria

* camaraproject/EdgeCloud/issues/213
* camaraproject/EdgeCloud/issues/203

**Comment by hdamker** [comment]
@Kevsy would you like to transfer the issues from EdgeCloud into the SimpleEdgeDiscovery repository? 

As you have write access to both repositories you can do it yourself, using the "Transfer issue" on the right column. Links to the previous location will be automatically redirected to the new location, so no problems with broken links in minutes etc.

**Comment by Kevsy** [comment]
> @Kevsy `x-correlator` is still defined with `format: uuid` in v0.11.0-rc.1, but Commonalities removed this restriction in v0.4.0. So `x-correlator` can now be any string.

Thanks @eric-murray - raised in #33 for M4

**Comment by hdamker** [comment]
@Kevsy don't forget the fix for #33

**Comment by Kevsy** [comment]
> @Kevsy don't forget the fix for #33

Thanks - added to the scope above

**Comment by Kevsy** [comment]
Added the following issues to the scope.

* camaraproject/SimpleEdgeDiscovery#34
* camaraproject/SimpleEdgeDiscovery#35
* camaraproject/SimpleEdgeDiscovery#36

**Comment by hdamker** [comment]
@Kevsy When creating the release PR for M4, please set the version therein to 1.0.0, not 0.11.0 -- assuming that you still consider to release a stable version. It would be great if you would also update the title and target version field on https://wiki.camaraproject.org/display/CAM/simple-edge-discovery+v0.11.0 already now to have a clear status here.

**Comment by Kevsy** [comment]
HI Herbert -

> When creating the release PR for M4, please set the version therein to 1.0.0, not 0.11.
Sure - just to clarify, you are not asking me to change the current PR for v0.11.0-rc.2<https://github.com/camaraproject/SimpleEdgeDiscovery/pull/37> - but rather to make sure the eventual release PR for M4 has v1.0.0 , correct?

> It would be great if you would also update the title and target version field on https://wiki.camaraproject.org/display/CAM/simple-edge-discovery+v0.11.0 already now to have a clear status here.

Yes, I can change that now

All best,
Kevin



C2 General

**Comment by Kevsy** [comment]
> @Kevsy When creating the release PR for M4, please set the version therein to 1.0.0, not 0.11.0 -- assuming that you still consider to release a stable version. 

Sure - just to clarify, you are not asking me to change the current PR #37 for v0.11.0-rc.2 - but rather to make sure the eventual release PR for M4 states  uses v1.0.0 , correct?

> It would be great if you would also update the title and target version field on https://wiki.camaraproject.org/display/CAM/simple-edge-discovery+v0.11.0 already now to have a clear status there.

Yes, I can change that now

**Comment by hdamker** [comment]
> Sure @hdamker - just to clarify, you are not asking me to change the current PR #37 for v0.11.0-rc.2 - but rather to make sure the eventual release PR for M4 states uses v1.0.0 , correct?

Yes, correct. We should just get #37 merged soon, as the "eventual release PR for M4" is already due on Monday :-)

**Comment by Kevsy** [comment]
Thanks @hdamker - so I can merge #37 now, release with tag r1.2, and then prepare the release PR for M4 by the Monday deadline :)

**Comment by hdamker** [comment]
@Kevsy thanks for https://github.com/camaraproject/SimpleEdgeDiscovery/pull/38 ... so you are now ready to create the release with the r1.2 tag 👍

**Comment by Kevsy** [comment]
@hdamker  Thanks, my mistake 🤦‍♂️ - also thanks to @javierlozallu for the fast approval.

I will make the release and then prepare the PR for M4 per the instructions - Monday is a UK bank holiday so will get it done today...

**Comment by hdamker** [comment]
@Kevsy I suppose that this issue is done and can be closed. Consider to start planning for Spring25 😄

**Comment by Kevsy** [comment]
Thanks Herbert and all the contributors for getting this issue closed :)
